users/views.py

```diff
@@ -66,14 +67,13 @@ def post(self, request):
 
                 token = RefreshToken.for_user(custom_user).access_token
                 current_site = get_current_site(request).domain
-                relative_link = reverse('send_email_confirmation')
-                abs_url = 'http://' + current_site + relative_link + '?token=' + str(token)
+                relative_link = reverse('verify-email', kwargs={'token': token, 'user_id': custom_user.id})
+                abs_url = 'http://'+ current_site + relative_link + '?token=' + str(token) + '?id=' + str(custom_user.id) 
```

Comment:
Look, maybe we shoul use `&` for separate parameters.

Reply:
In my another PR I already changed abs_url
![image](https://github.com/Project-Stage-Academy/UA_1155_alpha/assets/133953467/16485dbe-41d0-4d9b-b50e-f7cd5353f130)

Reply:
Oh, see in future PR we change it to f-string method🙌🏻✅